Tasks/XamarinAndroid/task.json

```diff
@@ -54,35 +53,48 @@
             "name": "outputDir",
             "type": "string",
             "label": "Output Directory",
-            "defaultValue": "$(Agent.BuildDirectory)\\bin\\Release",
+            "defaultValue": "",
             "required": false
         },
         {
             "name": "configuration",
             "type": "string",
             "label": "Configuration",
-            "defaultValue": "Release",
+            "defaultValue": "",
             "required": false
         },
-        { 
-            "name": "msbuildLocation", 
-            "type": "string", 
-            "label": "MSBuild Location", 
-            "defaultValue":"", 
+        {
+            "name": "msbuildLocation",
+            "type": "string",
+            "label": "Location",
```

Comment:
What do you think about "Build Tool Location"?  "Location" isn't very descriptive.